react-components/src/components/RevealToolbar/SettingsButton.tsx

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { useState, type ReactElement } from 'react';
+import { Button, Dropdown } from '@cognite/cogs.js';
+import { SettingsContainer } from './SettingsContainer/SettingsContainer';
+
+type CustomSettingsProps = {
+  customSettingsContent?: ReactElement;
+};
+
+export const SettingsButton = ({ customSettingsContent }: CustomSettingsProps): ReactElement => {
+  const [, setSettingsEnabled] = useState(false);
+  const [isHighQualityMode, setHighQualityMode] = useState(false);
```

Comment:
Any reason this cannot be in the SettingsContainer?

Reply:
https://github.com/cognitedata/reveal/pull/3578#discussion_r1294282777